rest/bootstrap_test.go

```diff
@@ -65,17 +65,16 @@ func TestBootstrapRESTAPISetup(t *testing.T) {
 	resp = bootstrapAdminRequest(t, http.MethodGet, "/db1/", ``)
 	assert.Equal(t, http.StatusOK, resp.StatusCode)
 	var dbRootResp DatabaseRoot
-	require.NoError(t, base.JSONDecoder(resp.Body).Decode(&dbRootResp))
-	require.NoError(t, resp.Body.Close())
+	resp.requireStatus(http.StatusOK)
```

Comment:
Is there a reason line 57 and 63 and other similar cases did not get changed to `resp.RequireStatus`?

It wouldn't cause any issues if the assert failed and the test continues, however if the initial PUT fails then the whole test is definitely going to fail.

Reply:
Just missed them in the find/replace

All of them should be caught now